tajo-dist/pom.xml

```diff
@@ -154,9 +154,6 @@
                       run cp -r ${project.basedir}/src/main/conf .
                       run rm -rf lib/tajo-*-${project.version}.jar
 
-                      run mkdir hive
```

Comment:
It should be kept because org.apache.hadoop.hive.ql.metadata.Table::getSd() included in hive-apache-0.9 causes NoSuchMethod exception.